File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int j = 0;
        int prev = -111;

        for(int i = 0; i < nums.size(); i++) {
            if(nums[i] != prev) {
                prev = nums[i];
                swap(nums[i], nums[j++]);
            }
        }

        return j;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.ts
function removeDuplicates(nums: number[]): number {
    
    let size = nums.length;
    let curr = 0;
    for(let i = 1; i < size; i++) {
        if(nums[curr] == nums[i]) {
            continue;
        } else {
            curr++;
            nums[curr] = nums[i];
        }
    }

    return curr + 1;
};